pkg/api/types.go

```diff
@@ -504,20 +506,32 @@ func (l *VolumeList) Set(value string) error {
 	if len(value) == 0 {
 		return errors.New("invalid format, must be source:destination")
 	}
+
+	var pos int
+	if isWindowsRegExp.MatchString(value) {
+		// We need to skip the 'C:\' that prefixes Windows directories, so we only
+		// check for ':' after that prefix.
+		pos = strings.Index(value[3:], ":") + 3
+	} else {
+		pos = strings.Index(value, ":")
```

Comment:
? i'm not sure what I'm supposed to be checking. I'm changing the code to grab the first ':' so that it preserves any possible file mode flags that could suffix the destination directory. Am I missing something? All of the tests pass (besides the added ones)